src/utils/prices.ts

```diff
@@ -85,6 +85,12 @@ const IMPACT_TIERS = [
 type WarningSeverity = 0 | 1 | 2 | 3 | 4
 export function warningSeverity(priceImpact: Percent | undefined): WarningSeverity {
   if (!priceImpact) return 0
+  // This function is used to calculate the Severity level for % changes in USD value and Price Impact.
+  // Price Impact is always an absolute value (conceptually always negative, but represented in code with a positive value)
+  // The USD value change can be positive or negative, and it follows the same standard as Price Impact (positive value is the typical case of a loss due to slippage).
```

Comment:
hm this is wrong btw, these values have nothing to do with slippage

Reply:
this is also confusing because the "standard practice" for price impact is an absolute value